plugins/org.python.pydev/src/org/python/pydev/ui/dialogs/PyDialogHelpers.java

```diff
@@ -161,6 +170,42 @@ public static int openQuestionConfigureInterpreter(AbstractInterpreterManager m)
         return INTERPRETER_CANCEL_CONFIG;
     }
 
+    /**
+     * <p>Statically open a list dialog to choose an interpreter from Conda.</p>
+     * <p>It returns both the interpreter name and executable.</p>
+     * 
+     * @return NameAndExecutable
+     */
+    public static NameAndExecutable openCondaInterpreterSelection() {
+        List<String> envs = CondaPackageManager.listCondaEnvironments(PyDevCondaPreferences.getExecutable());
+        List<String> envNames = new ArrayList<String>();
+
+        for (String env : List.copyOf(envs)) {
+            envNames.add(env.substring(env.lastIndexOf('\\') + 1));
+        }
+
+        String title = "Conda interpreter selection";
+        String message = "Select a intepreter from the list.";
+        Shell shell = EditorUtils.getShell();
+        ListDialog d = new ListDialog(shell);
+        d.setInput(envNames);
+        d.setContentProvider(new ListContentProvider());
+        d.setLabelProvider(new LabelProvider());
+        d.setMessage(message);
+        d.setTitle(title);
+        d.open();
+        if (d != null) {
+            String r = d.getResult()[0].toString();
+            String ret = envs.get(envNames.indexOf(r));
+            if (new File(ret + "\\python.exe").exists()) {
+                return new NameAndExecutable(r, ret + "\\python.exe");
+            } else if (new File(ret + "\\python.jar").exists()) {
```

Comment:
You don't need to check for `python.jar` -- usually it'd be `jython.jar`, but in this case we don't need to check for `jython` at all.